middleware/redirects.global.ts

```diff
@@ -15,36 +16,47 @@ export default defineNuxtRouteMiddleware((route) => {
     }
   }
 
-  let redirectValue
+  let redirectValue: RouteLocationRaw | null | undefined
 
   const paths = [
     {
-      cond: val => val === '/drops',
+      cond: (val: string) => val === '/drops',
       replaceValue: '/ahp/drops',
     },
     {
-      cond: val =>
+      cond: (val: string) =>
+        val.startsWith(`/${urlPrefix.value}`) && val.endsWith('profile'),
```

Comment:
```suggestion
        val === `/${urlPrefix.value}/profile`,
```
how about it?

Reply:
what about trailing slash? I don't think it will work as before